src/app/pregnancies/page.tsx

```
@@ -1,17 +1,64 @@
+          <div className="flex items-start gap-4">
+            <Calendar className="mt-0.5 size-6 text-blue-600" />
+            <div>
+              <p className="font-semibold text-blue-900">Ready to start tracking?</p>
+              <p className="mt-1 text-sm text-blue-800">Once your clinic connects your pregnancy record, you'll see detailed tracking and progress here.</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section className="rounded-3xl border border-[#F0F6F6] bg-white p-6 shadow-sm">
+        <h3 className="text-lg font-semibold text-[#1D5052]">What you'll track</h3>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)